docs/using_recompiled_libc.md

```diff
@@ -124,13 +128,16 @@ cd ~/rpmbuild/SPECS && rpmbuild -ba glibc.spec
 
 ## Using the compiled libc
 
-Our patched libc can be injected via `LD_LIBRARY_PATH` or `LD_PRELOAD`.
-I've been using `LD_LIBRARY_PATH` because there are actually multiple libraries
-compiled, and we'll want those others to be preferred as well (e.g. `libpthread`).
-In shadow's configuration file you can set the environment variable for all
-loaded plugins with the `environment` attribute of the `shadow` tag. e.g.:
+The patched libc can be injected into a plugin by setting `LD_LIBRARY_PATH` in each target plugin's environment. e.g.:
 
 ```
-<shadow environment="LD_LIBRARY_PATH=/path/to/glibc-2.27/build-tree/amd64-libc">
+...
+hosts:
+  my_host:
+    processes:
+      - path: /path/to/process
+      - environment: LD_LIBRARY_PATH=/path/to/glibc-2.27/build-tree/amd64-libc
```

Comment:
There shouldn't be a `-` before environment (they're part of the same map/dictionary).

Reply:
Fixed